bluesky/callbacks/best_effort.py

```diff
@@ -23,7 +23,7 @@
 
 class BestEffortCallback(QtAwareCallback):
     def __init__(self, *, fig_factory=None, table_enabled=True, **kwargs):
-        super().__init__(**kwargs)
+        super().__init__(use_teleporter=kwargs.pop('use_teleporter', None))
```

Comment:
I think this means we now accept kwargs but don't do anything with them (which can let typos silently pass).  Do we need to do this popping?

Reply:
Yes, I think you are right. I followed the pattern used in LivePlot, but BestEffortCallback treats the parameters differently.

Reply:
Indeed, the `kwargs` are not used anywhere inside the body of `BestEffortCallback`, which means we are just silently swallowing arbitrary parameters. Maybe while we're here we just remove `**kwargs` from the signature and go with:

```py
def __init__(self, *, fig_factory=None, table_enabled=True, use_teleporter=None):
    super().__init__(use_teleporter=use_teleporter)
```

Would that make sense?

Reply:
Ah, I just saw 7f943e7. That works too. Merging!